api/envoy/service/auth/v2/attribute_context.proto

```diff
@@ -116,6 +116,15 @@ message AttributeContext {
 
     // The HTTP request body.
     string body = 11;
+
+    // Specifies whether the body contains partial data to fit within the specified
+    // :ref:`max_request_bytes
+    // <envoy_api_field_config.filter.http.ext_authz.v2.BufferSettings.max_request_bytes>`
+    // setting when
+    // :ref:`allow_partial_message
+    // <envoy_api_field_config.filter.http.ext_authz.v2.BufferSettings.allow_partial_message>`
+    // is true.
+    bool partial_body = 12;
```

Comment:
Ah, yeah, I totally missed this. This complicates things as it seems this extra field does not translate to a plain HTTP request. I suppose it could be added as a header (`x-envoy-partial-body: 0|1`)?